tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -331,7 +336,7 @@ sub setup_longhorn_csi {
     assert_script_run(qq(kubectl patch -n longhorn-system lhs storage-minimal-available-percentage --type merge -p '{"value": "5"}'));
 
     # Create storage classes
-    assert_script_run("kubectl apply -f https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/main/storage/longhorn-sc.yaml");
+    assert_script_run("kubectl apply -f https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/c41969bca710335f7966b1588abca9958a33ff24/storage/longhorn-sc.yaml");
```

Comment:
I guess this commit will deem to change in future. Maybe better to put it into a testsuite variable which is configurable when launching test runs and will save the code change when the value needs to be updated.

Reply:
No, this is a permalink instead of 'main' which suggested by developer. You can treat this hash value as a fixed value for the file in git.

Reply:
Yes, that's how I understood it, so same with you. Someday that fixed shasum  will change to another one , and with current code you will need test code change again. My suggestion was that putting this shasum to a testsuite var, will save your code change here, and then only testsuite setting change will be enough.